doc/developer/projectStructure.md

```diff
@@ -0,0 +1,46 @@
+# Project Structure
+
+:movie_camera: [You can find code dive session recorede video here](https://youtu.be/_adhRj19ESY)
+
+```
+|_ doc
+|_ eng
+|_ examples
+|_ src
+    |_ Core
+    |   |_ ServiceWrapper
+    |       |_ Main.cs
+    |   |_ WinSWCore
+    |       |_ ServiceDescriptor.cs
+    |       |_ Configurations
+    |       |_ Extensions
+    |_ Plugins
+    |_ Test/winswTest
+```
+
+## :open_file_folder: examples
+
+This folder contains templates for configuration files. `sample-minimal.xml` contains a template for mandotory configurations and `sample-allOptions.xml` contains all possible configurations with documentation.
+
+## :open_file_folder: Core
+
+## :notebook: ServiceWrapper
+
+This is the main executable project. This contains the main.cs which is the entry point of the projet and contains some features for logging.
+
+### :page_facing_up: Main.cs
+
+This file contains the entry point of the program (Main method). This file includes the main flow of the program and has implemented the logics for following command line arguments.  
+ ```install, uninstall, start, stop, stopwait, restart, restart!, status, test, testwait, help, version```
```

Comment:
Rendering in GitHub editor went crazy there. Would it make sense to just reference documentation without providing a full list which will be obsolete soon anyway?

Reply:
Sure... Thank you for mention that !

Reply:
Sure. Thank you for mention that!

Reply:
Developer Guide is not working in `readme.md`. Shall I remove that as well.